example/test_tetanus.py

```diff
@@ -20,6 +20,7 @@ class TestsSpecGiraffe:
     _giraffe: Giraffe = None # type: ignore
 
     @classmethod
+    @pytest.mark.lego('giraffe.bob')
```

Comment:
Does it work via inheritance?

Reply:
If there is no setup_class then the setup_class method of the base class will be called.

If setup_class is defined in the child class, it can use lego mark and get different components.
Then it can call super().setup_class(components) and pass the desired components. 
So if you want to, you could define different components in the child test class.

Not sure if that was what you asked about 😅